Documentation/distributions.md

```diff
@@ -0,0 +1,50 @@
+# How to get rkt on each Linux distribution
+
+## CoreOS
+
+rkt is pre-installed on CoreOS.
+The [CoreOS release page](https://coreos.com/releases/) specifies the version of rkt available in each CoreOS release channel.
+
+## Fedora
+
+rkt is packaged in Fedora Rawhide:
```

Comment:
May want to note that rawhide is currently the development version and does go GA till midyear.

If you're on the current fedora stable, it far easier to install via the releases than to install the rawhide repos.